_posts/2016-06-22-Custom Font.md

```diff
@@ -64,8 +62,8 @@ to apply the font.
 
 ## Change Font Size
 
-A quicker way to change font size is to do that in preferences panel. On macOS, "font size" is in "General" section of preferences panel. On Window/Linux, it is under "Appearance" section, you also use press <kbd>Ctrl+F</kbd> to search it in preferences panel.
+A quicker way to change font size is in the preferences panel. On macOS, "font size" is in "General" section of preferences panel. On Window/Linux, it is under "Appearance" section, you also use press <kbd>Ctrl+F</kbd> to search for it in preferences panel.
 
-This option requires support from themes you use. If you are making a theme css, use `rem` as font unit will make sure this option works.
+This option requires support from the theme(s) you use. If you are making a theme css, use `rem` as the font unit to make sure this option works.
 
-<img src="/media/custom-font/Screen Shot 2019-01-25 at 22.18.21.png" style="zoom:50%" />
\ No newline at end of file
+![2](/media/custom-font/2.png)
```

Comment:
You could use `<img alt="2" src="/media/custom-font/2.png" style="zoom:50%" />` instead to keep the "zoom:50%" style. The screenshot is captured on macOS Retina screen, which is 2x larger than the actual size

Reply:
The <img...> syntax doesn't render in my code editor of choice (Visual Studio Code) whereas my alternative does. But I have reverted to your suggestion.